changelog/unreleased/issue-3456

```diff
@@ -0,0 +1,7 @@
+Enhancement: match untagged snapshots only when listing or forgetting
```

Comment:
This description is a bit unclear, might be better with "Support listing and forgetting untagged snapshots".